modules/calib3d/src/fisheye.cpp

```diff
@@ -253,7 +253,11 @@ void cv::fisheye::projectPoints(InputArray objectPoints, OutputArray imagePoints
 
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////
 /// cv::fisheye::distortPoints
-
+// Note:
+//   The input is normalized points and the output is raw points.
+//   The inputs must have the optical center subtracted
+//   and then divided by focal length.
+//   While the output is multiplied by focal length with optical center added.
```

Comment:
It's equivalent to the documentation note: `Note that the function assumes the camera intrinsic matrix of the undistorted points to be identity. This means if you want to distort image points you have to multiply them with or use another function overload.` See https://docs.opencv.org/4.x/db/d58/group__calib3d__fisheye.html#ga75d8877a98e38d0b29b6892c5f8d7765
Also I just added overload to handle undistort camera matrix: https://github.com/opencv/opencv/pull/25943. I do not think that the note is useful.

Reply:
Wait so are you saying this change I made isn't needed and we should close the reported issue?

Reply:
yes